src/sql/src/plan/statement/ddl.rs

```diff
@@ -4717,10 +4711,6 @@ pub fn plan_alter_item_set_cluster(
 
     match resolve_item_or_type(scx, object_type, name.clone(), if_exists)? {
         Some(entry) => {
```

Comment:
I'm a bit confused, where from this function do we actually return a `MismatchedIObjectType` error?

Reply:
not sure if this comment was resolved but the corresponding change is in `src/sql/src/plan/statement/ddl.rs`